src/services/FlawService.ts

```diff
@@ -128,7 +128,7 @@ export async function postFlawCvssScores(flawId: string, cvssScoreObject: unknow
     method: 'post',
     url: `/osidb/api/v1/flaws/${flawId}/cvss_scores`,
     data: postObject,
-  }, { beforeFetch })
+  })
```

Comment:
I think this bug was caused by the changes to the flaw Id fetching--are we certain that removing this doesn't lose functionality we need in order to avoid the error related to outdated `updated_dt` that the original `beforeFetch` addressed>

Reply:
Just thinking out loud. Does it make sense to check for `updated_dt` on POSTs? You are creating the resource and thus there is not such  `updated_dt` which you could compare to determine the freshness of data. Maybe the parent resource `updated_dt` should be checked in this case? That would at least make sense for prod/stage where Bugzilla sync is turned on and every child resource is in sync with the parent on `updated_dt`, for UAT, this is however not a case

Reply:
Yes, this bug appeared after the changes on how OSIM handles `updated_dt`.

Before was being sent the flaw's `updated_dt`, and now the `updated_dt` is fetched from OSIDB for the specific object of the operation and added to the operation's request.

The bug comes for that fetch attempt of an `updated_dt` for an object that doesn't exists.
As this is a POST method used exclusively for `cvss_scores` creation the `updated_dt` is not needed (OSIDB doesn't expects/handles it)

Apart from the theory I have tested around I didn't found any lost feature or new bugs.

Reply:
> Is there or would there ever be a situation where the flaw's updated_dt becomes out of date in between loading the flaw and saving the CVSS score?
I think that scenario is the same as two users are in the same flaw and one edits the CVSS before the other, the second one will get outdated model instance. But I see it as the normal behavior on the updated_dt purprose.